flang/docs/OpenACC.md

```diff
@@ -25,3 +25,10 @@ local:
   logical expression.
 * `!$acc routine` directive can be placed at the top level. 
 * `!$acc cache` directive accepts scalar variable.
+
+## Remarks about incompatibilities with other implementations
+* Array element references in the data clauses are equivalent to array sections
+  consisting of this single element, i.e. `copyin(a[n])` is equivalent to
```

Comment:
Shouldn't we use the parenthesis notation instead of brackets for Fortran?

Reply:
Thanks for the catch!